src/actions/account.js

```diff
@@ -42,6 +33,7 @@ export function handleRefreshAccount(history, loader = true) {
                   accounts: wallet.accounts
                }
             })
+            dispatch(loginSuccess(true))
          })
          .catch(e => {
             console.error('Error loading account:', e)
```

Comment:
I think stuff in ` if (e.message && e.message.indexOf('does not exist while viewing') !== -1) {` block should be moved into separate RESET_ACCOUNTS action.